SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3531,6 +3533,136 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 	return false;
 }
 
+void USpatialNetDriver::ServerReplicateActors_BuildConsiderList(TArray<FNetworkObjectInfo*>& OutConsiderList, const float ServerTickTime)
+{
+	TArray<FNetworkObjectInfo*> TmpConsiderList;
+	TmpConsiderList.Reserve(GetNetworkObjectList().GetActiveObjects().Num());
+
+	Super::ServerReplicateActors_BuildConsiderList(TmpConsiderList, ServerTickTime);
+
+	// Cache if we are running with Spatial Networking to avoid getting the settings in every iteration of the loop.
+	const bool bSpatialNetworking = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+	if (!bSpatialNetworking)
+	{
+		return;
+	}
+
+	for (FNetworkObjectInfo* const& ActorInfo : TmpConsiderList)
+	{
+		AActor* Actor = ActorInfo->Actor;
+		check(Actor != nullptr);
+
+		Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(Actor);
```

Comment:
Since you don't want the overhead of this call if the Actor does not have authority, I'd move this function into the function "CheckUnauthorisedDataChanges"

Reply:
Moved it inside CheckUnauthorisedDataChanges.